### ISSUE TITLE
CNN Optimize number of training iterations

### ISSUE THREAD

**Issue by tiimoS** [literally-a-placeholder/pattern_recognition#21]
Optimize number of training iterations. Plot a graph showing the accuracy on the training
set and the validation set, respectively, with respect to the training epochs.

**Comment by tiimoS** [comment]
should be around 25 as far as I've heard from colleagues
